common/lib/api-client/middleware/response.js

```diff
@@ -0,0 +1,48 @@
+const { isArray } = require('lodash')
+
+function needsDeserialization(method) {
+  return ['GET', 'PATCH', 'POST'].indexOf(method.toUpperCase()) !== -1
```

Comment:
Could we use [].includes here? So it would be `['GET', 'PATCH', 'POST'].includes(method.toUpperCase())`